pkg/hintrunner/zero/zerohint_signatures.go

```diff
@@ -81,7 +82,7 @@ func newGetPointFromXHinter(xCube, v hinter.ResOperander) hinter.Hinter {
 			}
 
 			//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack
-			secpBig, _ := secp_utils.GetSecPBig()
+			secpUint256 := secp_utils.GetSecPUint256()
```

Comment:
Regarding variable names here as well, I don't think we need to append the type behind it. We can explicitly declare their type if you think it is needed for readability.

Reply:
This happen overall in this function

Reply:
i remove it . For `vBig := v.BigInt(new(big.Int))`. i need to keep it as using `v` will not compile